src/vs/workbench/api/common/extHostMcp.ts

```diff
@@ -33,6 +33,7 @@ export interface IExtHostMpcService extends ExtHostMcpShape {
 
 export class ExtHostMcpService extends Disposable implements IExtHostMpcService {
 	protected _proxy: MainThreadMcpShape;
+	protected _mcpHttpHandleType = McpHTTPHandle;
```

Comment:
Let's keep this class unchanged and instead modify our override of `_startMcp` in the Node implementation of this service

Reply:
Done!